src/helpers/axios.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { currentUser, refreshUser } from 'redux/auth/operations';
+import { store } from 'redux/store';
+
+export const API = axios.create({
+  baseURL: '',
```

Comment:
add to baseURL value from constant or env server link: https://h2o-experts-server.onrender.com